UEBPrep/app/views/homepage/index.html.erb

```diff
@@ -1,24 +1,8 @@
-<div id="home-header">
-  <img id="home-logo" src="assets/homepage/logo2.png" alt="image">
+<div class="row">
+  <div id="home-content" class="small-12 small-centered columns">
+    <h1>Unified English Braille eLearning Platform</h1>
+  </div>
```

Comment:
where is the rest of the homepage content being placed? Or are you breaking it out into a partial?